resources/scss/modular/_our-reporting-grid.scss

```diff
@@ -106,6 +106,10 @@ $font-size: 80px;
             }
         }
 
+        .our-reporting-card__excerpt {
+            font-weight: 300;
+        }
+
```

Comment:
I would consider doing the opposite here, making the card content `font-weight: 300`, and then explicitly setting the excerpt to 300. My reasoning here is that in the future if we make the excerpt a WYSIWYG, bold text will likely not work correctly.